.github/workflows/bloc_test.yaml

```diff
@@ -24,11 +24,11 @@ jobs:
       - name: Install Dependencies
         run: pub get
       - name: Format
-        run: dartfmt --dry-run --set-exit-if-changed .
+        run: dart format --set-exit-if-changed none
```

Comment:
I believe `dart format` expects a directory so in this case the command does not format any files because a directory called `none` does not exist
```suggestion
        run: dart format --set-exit-if-changed .
```

![Screen Shot 2021-02-18 at 3 50 43 PM](https://user-images.githubusercontent.com/8855632/108426249-29fe5680-7201-11eb-92d2-574e5734f095.png)

Reply:
Nope)
One second.

Reply:
```bash
PS > dart format -h
Idiomatically format Dart source code.

Usage: dart format [options...] <files or directories...>
-h, --help                     Print this usage information.
-v, --verbose                  Show all options and flags with --help.
-o, --output                   Set where to write formatted output.

          [json]               Print code and selection as JSON.
          [none]               Discard output.
          [show]               Print code to terminal.
          [write] (default)    Overwrite formatted files on disk.

    --set-exit-if-changed      Return exit code 1 if there are any formatting changes.
    --fix                      Apply all style fixes.
-l, --line-length              Wrap lines longer than this.
                               (defaults to "80")

Run "dart help" to see global options.
```

You anyway can't change files under git.

With option
`[none] - Discard output.`
we only check format source code and throw error if something wrong.

Reply:
Also, syntax is changed.
If you anyway want `.`, probably you want `write` instead.

Reply:
@felangel sorry, my bad.  Must be: `dart format --set-exit-if-changed -o none .`